docs/input_tags.rst

```diff
@@ -532,6 +533,11 @@ SC.MetricFactor (*real*)
 
     *default*: 0.1
 
+SC.MakeInitialChargeFromK (*boolean*)
+    Flag determining whether initial charge is made from the density matrix
+
+    *default*: T
```

Comment:
Default value is T only when General.LoadDM is T. (General.LoadDM is T when AtomMove.ReuseDM is T.)